README.md

```diff
@@ -1,105 +1,60 @@
+**// Add screenshot 3 here //**
 
-```bash
-npm install
-```
+- The Sleep section will show you:
+    1. The amount of sleep you got last night and your average for the week.
+    2. A chart showing your sleep quality and amount for each day over the past week.
+    3. Your all time daily average for hours slept and sleep quality and your sleep ranking for the week.
+**// Add screenshot 4 here//**
 
-To verify that it is setup correctly, run `npm start` in your terminal. Go to `http://localhost:8080/` and you should see a page with some `h1` text and a pink background. If that's the case, you're good to go. Enter `control + c` in your terminal to stop the server at any time.
+- The Activity section will show you:
+    1. Activity today (steps, minutes active, and distance walked in miles and kilometers).
+    2. How much over or under the average you are vs. all users in steps, minutes active, and flights of stairs for the day.
+    3. You activity per day over the past week.
+**// Add screenshot 5 here//**
 
-## Where to Add Your Code
+### Setup
 
-### JavaScript
+- To run locally:
+    1. Clone and/or fork the repo.
+    2. Run `npm install`.
+    3. To launch, `npm run start` or `npm start` and copy the provided `localhost` url into your browser.
 
-You have to be very intentional with where you add your feature code. This repo uses a tool called [webpack](https://webpack.js.org/) to combine many JavaScript files into one big file. Webpack enables you to have many, separate JavaScript files to keep your code organized and readable. Webpack expects all of your code files to be in a specific place, or else it doesn't know how to combine them all behind the scenes.
+### Known Bugs/Issues
 
-**Create all of your feature code files in the `src` directory.**
+- When adding data in one of the forms on the main page an error message will display if you try to submit the form without filling out all of the fields. This error message will not go away until the missing fields are filled out **and** the form is submitted. The error should go away when all inputs are filled out but before the form is submitted.
 
-Since code is separated into multiple files, you need to use the `import` and `export` syntax to share code across file.
+- If a form element on the main page is highlighted because it was not filled out, entering an invalid character will cause the error highlighting to disappear. This should only happen if the input is valid.
 
-Here is a video that walks through some information about [import and export](https://www.youtube.com/watch?v=_3oSWwapPKQ). There are a lot of resources out there about `import` and `export`, and resources will sometimes call them `ES6 modules`. It's something you will see in React and beyond.
+- This app is not fully optimized for smaller screen sizes and you may experience issues when viewing on a smaller screen or window.
 
-### HTML
+- Our charts are not fully color blind accessible.
```

Comment:
Let's add something in here about specifically the patterns.  I would reword this to, "A future iteration to make our charts even more readable by someone who has color blindness would be to add a pattern to each individual part of the graph. At this point, the charts are readable by people who have color blindness, but we want to make the visual look even more differentiated."